### PR TITLE
[d16-8] Bring Xcode12.1 support.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: Make.config [hunk 1]
```diff
@@ -111,9 +111,9 @@ WATCHOS_NUGET_VERSION_NO_METADATA=$(WATCHOS_NUGET_VERSION)-$(NUGET_PRERELEASE_ID
 WATCHOS_NUGET_VERSION_FULL=$(WATCHOS_NUGET_VERSION_NO_METADATA)+$(NUGET_BUILD_METADATA)
 
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
-XCODE_VERSION=12.0
-XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_12.xip
-XCODE_DEVELOPER_ROOT=/Applications/Xcode_12.0.0.app/Contents/Developer
+XCODE_VERSION=12.1
+XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_12.1_GM_seed.xip
+XCODE_DEVELOPER_ROOT=/Applications/Xcode_12.1.0-GM.app/Contents/Developer
```
Comment (dalexsoto) [inline Make.config:116]: This is right because there is no package yer

File: Make.config [hunk 2]
```diff
@@ -35,7 +35,7 @@ $(TOP)/Make.config.inc: $(TOP)/Make.config $(TOP)/mk/mono.mk
 
 include $(TOP)/Make.versions
 
-APIDIFF_REFERENCES=https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/xcode12/5bcf5b877d2101a21f97b93d5c912273d804110d/238/package/bundle.zip
+APIDIFF_REFERENCES=https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/xcode12/7ec3751a1077e5d77d7375ef61182538d9d11f8d/234/package/bundle.zip
```
Comment (spouliot) [inline Make.config:38]: that should be the `.../xcode12.1/commit-you-merged/...`

**Review by spouliot — CHANGES_REQUESTED** [review]

**Review by dalexsoto — APPROVED** [review]

File: Make.config (hunk 2 above)
Comment (dalexsoto) [inline Make.config:38]:
```suggestion
APIDIFF_REFERENCES=https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/xcode12.1/e0ce496804e49056ab8eb41da55813083ab7f58b/15/package/bundle.zip
```

**Review by spouliot — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Review by whitneyschmidt — APPROVED** [review]

**Comment by dalexsoto** [comment]
`ARFrameSemantics` breaking change is expected and correct

**Pull request merged**